notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb

```diff
@@ -0,0 +1,301 @@
+   "source": [
+    "# Locate the file that has the data\n",
+    "want = (results['sectors'] == \"s0001-s0013\") & (results['fileType'] == \"dvt.fits\")\n",
+    "dvt_filename = manifest[want]['Local Path'][0]\n",
+    "\n",
+    "# Print out the file info\n",
+    "fits.info(dvt_filename)\n",
+    "\n",
+    "# Plot the first time series\n",
+    "data = fits.getdata(dvt_filename, 1)\n",
+    "time = data['TIME']\n",
+    "relflux = data['LC_DETREND']\n",
+    "\n",
+    "plt.figure(figsize=(16,3))\n",
+    "plt.plot (time, relflux, 'b.')\n",
+    "plt.ylim(1.2* np.nanpercentile(relflux, .5) , 1.2 * np.nanpercentile(relflux, 99.5))\n",
+    "plt.title('Data Validation Detrended Light Curve for %s' % (star_name))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot Folded Light curve\n",
+    "Each extension of the DVT data file contains a separate TCE. After the pipeline finds a set of transits, the transits are removed and the light curve is once again searched for transits.  L98-59 has three TCEs, each is consistent with the three confirmed planets found around this star.  Here we plot the phase folded light curve for each TCE, each as its own subplot.  The DVT file also contains a transit model as one of the columns in the FITS table. We overplot that in orange."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_folded(phase, data, model, ext, period):\n",
+    "    isort = phase.argsort()\n",
+    "    \n",
+    "    plt.plot(phase[isort], data[isort], '.', ms = .5)\n",
+    "    plt.plot(phase[isort], model[isort], '-', lw=1, label = \"TCE %i\" % ext)\n",
+    "    plt.xlabel('Phase (Period = %5.2f days)' % period)\n",
+    "    plt.ylim(1.5 * np.nanpercentile(data, .5) , 1.4 * np.nanpercentile(data,99.5))\n",
+    "    plt.legend(loc = \"lower right\")\n",
+    "\n",
+    "plt.figure(figsize = (14,10))\n",
+    "\n",
+    "nTCEs = fits.getheader(dvt_filename)['NEXTEND'] - 2\n",
+    "\n",
+    "for ext in range(1,nTCEs + 1):\n",
+    "    data = fits.getdata(dvt_filename, ext)\n",
+    "    head = fits.getheader(dvt_filename, ext)\n",
+    "    period = head['TPERIOD']\n",
+    "    phase = data['PHASE']\n",
+    "    flux = data['LC_INIT']\n",
+    "    model = data['MODEL_INIT']\n",
+    "    plt.subplot(3, 1, ext)\n",
+    "    plot_folded(phase, flux, model, ext, period)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"about_ID\"></a>\n",
+    "## About this Notebook\n",
+    "**Authors:** \n",
+    "<br>Susan E. Mullally, STScI \n",
+    "<br>**Updated On:** 2019-09-13"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Top of Page](#title_ID)\n",
+    "<img style=\"float: right;\" src=\"./stsci_pri_combo_mark_horizonal_white_bkgd.png\" alt=\"stsci_pri_combo_mark_horizonal_white_bkgd\" width=\"200px\"/> "
```

Comment:
Use this image call to get the logo to show up properly:
`<img style="float: right;" src="https://raw.githubusercontent.com/spacetelescope/notebooks/master/assets/stsci_pri_combo_mark_horizonal_white_bkgd.png" alt="STScI logo" width="200px"/> `